charts/redpanda/templates/statefulset.yaml

```diff
@@ -124,88 +123,20 @@ spec:
 {{- end }}
         - name: {{ (include "redpanda.name" .) | trunc 51 }}-configurator
           image: {{ .Values.image.repository }}:{{ template "redpanda.tag" . }}
-          command: ["/bin/bash", "-c"]
+          command:
+            - /bin/bash
+            - -c
+            - | 
+              /etc/secrets/configurator/scripts/configurator.sh "${SERVICE_NAME}" "${KUBERNETES_NODE_NAME}"
```

Comment:
Perhaps we should do this for every container that requires a shell script to ensure sigterm is caught and used:
```
         env:
           - name: SCRIPT
           - value: /etc/secrets/configurator/scripts/configurator.sh
          command:
            - /bin/sh
            - -c
            - 'trap "exit 0" TERM; exec $SCRIPT & wait $!'
```

Reply:
this is an init container, so should run only once right?

Reply:
Yes, but I'm just thinking we should do this as best practice. If a pod is terminated during init, there's no reason to be forced to wait for the script to finish.